### ISSUE TITLE
This is a joke, right ?

### ISSUE THREAD

**Issue by ghost** [BPI-SINOVOIP/BPI-Mainline-kernel#4]
Just taking a mainline kernel and calling it BPI is not much of an effort.
There's just the standard 'sunxi_defconfig' in arm/configs, and an original Mainline
set of DTB's.
All that's really needed are some DTB's from Allwinner, then you (BPI) can leave the rest to the community...
    -rasp

**Comment by parrotgeek1** [comment]
@rasp they put the DTB in their armbian fork instead of here, for some reason https://github.com/BPI-SINOVOIP/BPI-files/commit/4a955b91d05fc33099c2d5ed23257da351799f36

**Comment by ghost** [comment]
Thanks, bit I was actually looking for a r/v40 dtb/dts, which they simply don't provide.
Most of the dtb's are from the community written from soc manuals.
SONOVOIP is just about calling every other thing bananapi-something without providing support.
I'm currently stuck at not knowing whether the mac-driver just doesn't recognize the realtek phy or
my dts attempts are complete rubbish.
Same goes for usb-phy (PH23 'seems' to power the usb somehow, A17 restarts the ether-phy).
As for the schematic, on the phy(MAC-page) ENSWREG turns into EPHY-AVDD33 and gets lost somewhere between the MAC and the CPU or AXP sheet of the schematic......
(This is not personally addressed at you, of course :-) )

**Comment by sarnold** [comment]
Yeah, the latest emac driver is fubar'd on all of the H3/a83t stuff I have.  I grew tired of it and moved on to espressobin and imx6 udoo stuff...
